scripts/upgrade

```diff
@@ -45,13 +45,19 @@ if [ -z "$final_path" ]; then
 	ynh_app_setting_set --app=$app --key=final_path --value=$final_path
 fi
 
+# If language doesn't exist, create it
+if [ -z "$language" ]; then
+	language="en"
+	ynh_app_setting_set --app=$app --key=language --value=$language
+fi
```

Comment:
perfect, now would be a good idea to remove old `lang` var

Reply:
mmh ... I don't see where I left `lang`var behind ... What exactly should I remove?

Reply:
the `lang` var is in the settings of previous installed rainloop, when they will be updated, as you change the var name, you should remove it

Reply:
how do I do that?

Reply:
You can use: `ynh_app_setting_delete`

Reply:
Thanks